variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/AbstractVariant.java

```diff
@@ -181,6 +181,13 @@ public String getMainId() {
         return mainId;
     }
 
+    public void renormalizeVariant(long newStart, long newEnd, String newReference, String newAlternate) {
```

Comment:
I was thinking that maybe this could be called just `renormalize`, because when you call it will look something like:

```java
myVariant.renormalize(1, 2, "AC", "A");
```